Resources/Locale/en-US/game-ticking/game-presets/preset-nukeops.ftl

```diff
@@ -3,7 +3,7 @@ nukeops-description = Nuclear operatives have targeted the station. Try to keep
 
 nukeops-welcome =
     You are a nuclear operative. Your goal is to blow up {$station}, and ensure that it is nothing but a pile of rubble. Your bosses, the Syndicate, have provided you with the tools you'll need for the task.
-    Death to Nanotrasen!
+    Operation {$operation_name} is a go ! Death to Nanotrasen!
```

Comment:
`operation_name` looks long-ish, I'd just leave `name`. Don't forget to change it in the `Loc.GetString()` call
```suggestion
    Operation {$name} is a go! Death to Nanotrasen!
```